ResearchKit/Common/ORKPasscodeViewController.h

```diff
@@ -129,6 +129,17 @@ ORK_CLASS_AVAILABLE
  */
 + (BOOL)removePasscodeFromKeychain;
 
+/**
+ This method allows to force the passcode to be a certain value. You only want to use this capability when you're moving your existing users
+ from a custom passcode entry mechanism to the one provided by ResearchKit.
+ 
+ This method will raise an exception if `passcode` is nil.
+ 
+ @param passcode          The passcode to store
+ @param touchIdEnabled    Whether TouchId will be available during passcode entry
+ */
++ (void)forcePasscode:(NSString *)passcode withTouchIdEnabled:(BOOL)touchIdEnabled;
```

Comment:
I need to call this method from outside of ResearchKit, at startup in the app delegate, where it detects the update that switched to RK's passcode entry. So there needs to be a public method somewhere, just like `removePasscodeFromKeychain` which already lives in this class. Where should it be moved to?